db/seeds.rb

```diff
@@ -31,7 +34,22 @@ def create_stickers
                                       image_url: "http://devstickers.com/assets/img/cat/chrome.png",
                                       price: 2,
                                       description: "Chrome logo")
-    puts "#{Sticker.all.map(&:title).join(", ")} created."
+    puts "#{Sticker.all.map(&:title).join(", ")} stickers created."
```

Comment:
Prefer single-quoted strings inside interpolations.<br>Do not write to stdout. Use Rails' logger if you want to log.